### PR TITLE
feat: add total distance in log message

### LINKED ISSUE #3203: Include total distance in `Downloading bodies` log message
### Describe the feature

atm the log message reads like:

> Downloading bodies 3701342..=3703534 

It would be helpful to also include the distance of the range `distance = end - start` so we can see the number of blocks directly

https://github.com/paradigmxyz/reth/blob/b90d0b41069015d5746b08b3e408e738bc65a6a7/crates/net/downloaders/src/bodies/bodies.rs#L335-L336

and 
https://github.com/paradigmxyz/reth/blob/b90d0b41069015d5746b08b3e408e738bc65a6a7/crates/net/downloaders/src/bodies/bodies.rs#L343-L344

wdyt @shre

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by yuichiroaoki**
resolve #3203

### CODE REVIEW

File: crates/net/downloaders/src/bodies/bodies.rs [hunk 1]
```diff
@@ -329,18 +329,19 @@ where
 
         // Check if the provided range is the next expected range.
         let is_next_consecutive_range = *range.start() == *self.download_range.end() + 1;
+        let total_distance = *range.end() - *range.start();
         if is_next_consecutive_range {
             // New range received.
             tracing::trace!(target: "downloaders::bodies", ?range, "New download range set");
-            info!(target: "downloaders::bodies", "Downloading bodies {range:?}");
+            info!(target: "downloaders::bodies", "Downloading bodies {range:?} (total distance: {total_distance})");
```
Comment (mattsse) [inline crates/net/downloaders/src/bodies/bodies.rs:336]: `distance` should be sufficient
```suggestion
            info!(target: "downloaders::bodies", "Downloading bodies {range:?} (distance: {total_distance})");
```

File: crates/net/downloaders/src/bodies/bodies.rs [hunk 2]
```diff
@@ -329,18 +329,19 @@ where
...
             self.download_range = range;
             return Ok(())
         }
 
         // The block range is reset. This can happen either after unwind or after the bodies were
         // written by external services (e.g. BlockchainTree).
         tracing::trace!(target: "downloaders::bodies", ?range, prev_range = ?self.download_range, "Download range reset");
-        info!(target: "downloaders::bodies", "Downloading bodies {range:?}");
+        info!(target: "downloaders::bodies", "Downloading bodies {range:?} (total distance: {total_distance})");
```
Comment (mattsse) [inline crates/net/downloaders/src/bodies/bodies.rs:344]:
```suggestion
        info!(target: "downloaders::bodies", "Downloading bodies {range:?} (distance: {total_distance})");
```

**Review by mattsse — CHANGES_REQUESTED** [review]
smol nit,

pending @shekhirin

**Comment by shekhirin** [comment]
@yuichiroaoki @mattsse how do you feel about including this as a logging field, and not part of the message?

**Comment by mattsse** [comment]
like

```
info!(target: "downloaders::bodies", %distance, "Downloading bodies {range:?}");
```

?

yeah that's actually better

**Review by mattsse — APPROVED** [review]
ty

**Pull request merged**